axelor-business-support/src/main/resources/domains/ProjectCategory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<domain-models xmlns="http://axelor.com/xml/ns/domain-models"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="http://axelor.com/xml/ns/domain-models http://axelor.com/xml/ns/domain-models/domain-models_5.0.xsd">
+
+	<module package="com.axelor.apps.project.db" name="project"/>
+
+	<entity name="ProjectCategory">
+		<integer name="defaultInvoicing" title="Default Invoicing" selection="support.default.invoicing.select"/>
```

Comment:
set default=1 for defaultInvoicing, else it will create issue. Also provide sql script to set default value.